modules/tab/Tab.tsx

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import { StandardProps, WithStylesProps, ThemeStyleSheetFactory } from "@diana-ui/types";
 import { withStyles } from "@diana-ui/base";
+import { Flipped } from "react-flip-toolkit";
```

Comment:
not that it matters, but could we have this reorder to the second line? just to have the diana imports all together